e2e/tests/node_reboot.robot

```diff
@@ -12,25 +18,30 @@ Test Teardown    Cleanup test resources
 ${LOOP_COUNT}    1
 ${RETRY_COUNT}    300
 ${RETRY_INTERVAL}    1
-${VOLUME_TYPE}    rwo
+${VOLUME_TYPE}    RWO
 ${CONTROL_PLANE_NODE_NETWORK_LATENCY_IN_MS}    0
 
 *** Test Cases ***
 Reboot Node One By One While Workload Heavy Writing
```

Comment:
Test case `Reboot Node One By One While Workload Heavy Writing` failed with error message:
```
ApiException: (0)
Reason: Handshake status 500 Internal Server Error -+-+- {'content-length': '29', 'content-type': 'text/plain; charset=utf-8', 'date': 'Tue, 05 Mar 2024 11:59:18 GMT'} -+-+- b'container not found ("sleep")'
```
Could you help check this? Thank you!
https://ci.longhorn.io/job/private/job/longhorn-e2e-test/375/console

Reply:
Thanks, should be fixed now.